neonvm-controller/deployment.yaml

```diff
@@ -55,6 +55,7 @@ spec:
         - "--concurrency-limit=128"
         - "--skip-update-validation-for="
         - "--disable-runner-cgroup"
+        - "--use-cpu-sysfs-state-scaling"        
```

Comment:
We probably shouldn't enable by default on initial rollout, right? otherwise rollback will result in new computes being unable to be handled by the old controller (?)